.github/workflows/industrial_ci_action.yml

```diff
@@ -0,0 +1,36 @@
+# This config uses industrial_ci (https://github.com/ros-industrial/industrial_ci.git).
+# For troubleshooting, see readme (https://github.com/ros-industrial/industrial_ci/blob/master/README.rst)
+
+name: BuildAndTest
+
+# This determines when this workflow is run
+on:
+  # When ros2 branch is pushed to
+  push:
+    branches: [ ros2 ]
+  # When there is a pull request against ros2
+  pull_request:
+    branches: [ ros2 ]
```

Comment:
I guess this can be simplified: On other branches, the workflow isn't defined at all (or differently).
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-using-a-list-of-events
```suggestion
# Triggers the workflow on push or pull request events
on: [push, pull_request]
```